chart/kubeapps/templates/mongodb-secret-bootstrap.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ template "kubeapps.mongodb-secret-bootstrap.fullname" . }}
+  annotations:
+    helm.sh/hook: pre-install
+    helm.sh/hook-delete-policy: hook-succeeded
+  labels:
+    app: {{ template "kubeapps.mongodb-secret-bootstrap.fullname" . }}
+    chart: {{ template "kubeapps.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  template:
+    metadata:
+      labels:
+        app: {{ template "kubeapps.mongodb-secret-bootstrap.fullname" . }}
+        release: {{ .Release.Name }}
+    spec:
+      containers:
+      - name: kubectl
+        image: {{ template "kubeapps.image" .Values.hooks.image }}
+        command:
+        - kubectl
+        - create
```

Comment:
what happens if the secret happens to exist? this will fail?

Reply:
yes, if the secret already exists the job execution will fail but the hook will success because it can create the job. Because of that the job will be deleted.